README.md

```diff
@@ -90,7 +90,9 @@ You can deploy this project using:
 ## 🔭 Future Features <a name="future-features"></a>
 
 
-#TBA#
+# Validations
+
+# Model specs
```

Comment:
### Optional
- [ ] To properly format the README file, let us kindly change `# Validations` to ` - [ ] Validations`, kindly do same for the model specs feature